src/lib/styles/elements/breadcrumbs.css

```diff
@@ -0,0 +1,13 @@
+/* Tailwind Elements: breadcrumbs.css */
+
+@layer components {
+	.breadcrumbs {
+		@apply flex items-center space-x-4 w-full hide-scrollbar overflow-x-auto;
+	}
+	.crumb {
+		@apply flex justify-center items-center space-x-2;
+	}
+	.crumb-separator {
+		@apply flex text-surface-700-200-token w-2 opacity-50;
```

Comment:
small sanity check, should it be max-w or min-w? Likely not since this is pretty static but I figure it's worth double checking

Reply:
Most of these styles are remnant from the component. It shouldn't have a width at all! Removed!